.travis.yml

```diff
@@ -6,6 +6,7 @@ os:
 python:
   - 2.7
   - 3.6
+  - 3.7
```

Comment:
Why not 3.8?

Reply:
To be honest I did not check whether Travis now has 3.8 as standard. Is that the case? It would indeed be good to add support there too.

Reply:
Yep: https://docs.travis-ci.com/user/languages/python/

I would just have the earliest supported (let's say 3.6) and the latest (3.8), along with 2.7 for now.

Reply:
Agree on the various fronts. My blunt opinion is: at this point, get going with a non-fine-tuned first metapackage release and then smooth out everything else. There are many things to do to get to a stable alpha release. We can organise the effort with issues ...